cmd/release-controller/types.go

```diff
@@ -466,6 +466,9 @@ const (
 	// releaseAnnotationBugsVerified indicates whether or not the release has been
 	// processed by the BugzillaVerifier
 	releaseAnnotationBugsVerified = "release.openshift.io/bugs-verified"
+
+	// inform DPTP-controller-manager that the imagestreamtag should be deleted
+	releaseAnnotationSoftDelete = "ci.openshift.io/soft-delete"
```

Comment:
Why is this annotation not consistent with the release naming scheme?  If there is an argument for this being a different name please make it here in the comments.

Also, your godoc should be in the correct form (and DPTP is not something a comment cares about in this code, you might want to say "automation external to the release controller can use this annotation to decide when to clean up the tag")

Reply:
Updated.